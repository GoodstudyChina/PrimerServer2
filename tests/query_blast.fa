>P1.0.Primer0
CTTCTGCAATGCCAAGTCCAG
>P1.0.Primer1
GTGGTGAAGGGTCGGTTGAA
>P1.1.Primer0
ACCAAACCCCAGAGTCAATTAA
>P1.1.Primer1
TCTATCTATTGCACTGCCTGTTG
>P2.0.Primer0
TTCAACCGACCCTTCACCAC
>P2.0.Primer1
GCTGCTGATCCCATTGCTTC