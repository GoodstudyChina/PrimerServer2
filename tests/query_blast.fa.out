P1.0.Primer0	21	1	21	seq1	655	17	37	plus
P1.0.Primer0	21	4	10	seq1	655	161	155	minus
P1.0.Primer0	21	9	15	seq1	655	534	540	plus
P1.0.Primer0	21	4	10	seq2	564	124	118	minus
P1.0.Primer0	21	9	15	seq2	564	496	502	plus
P1.0.Primer1	20	1	20	seq2	564	301	282	minus
P1.0.Primer1	20	5	20	seq2	564	105	90	minus
P1.0.Primer1	20	2	8	seq2	564	54	48	minus
P1.0.Primer1	20	1	20	seq1	655	338	319	minus
P1.0.Primer1	20	5	20	seq1	655	142	127	minus
P1.0.Primer1	20	2	8	seq1	655	91	85	minus
P1.1.Primer0	22	1	22	seq2	564	65	86	plus
P1.1.Primer0	22	1	7	seq2	564	94	100	plus
P1.1.Primer0	22	1	22	seq1	655	102	123	plus
P1.1.Primer0	22	1	7	seq1	655	131	137	plus
P1.1.Primer1	23	1	23	seq1	655	534	512	minus
P1.1.Primer1	23	3	10	seq1	655	388	381	minus
P1.1.Primer1	23	7	13	seq1	655	154	160	plus
P1.1.Primer1	23	1	23	seq2	564	496	474	minus
P1.1.Primer1	23	3	10	seq2	564	351	344	minus
P1.1.Primer1	23	7	13	seq2	564	117	123	plus
P2.0.Primer0	20	1	20	seq2	564	282	301	plus
P2.0.Primer0	20	1	16	seq2	564	90	105	plus
P2.0.Primer0	20	13	19	seq2	564	48	54	plus
P2.0.Primer0	20	1	20	seq1	655	319	338	plus
P2.0.Primer0	20	1	16	seq1	655	127	142	plus
P2.0.Primer0	20	13	19	seq1	655	85	91	plus
P2.0.Primer1	20	1	20	seq2	564	462	443	minus
P2.0.Primer1	20	1	20	seq1	655	500	481	minus
P2.0.Primer1	20	1	7	seq1	655	41	35	minus
